Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -296,6 +298,17 @@ YUI.add('ez-locationviewlocationstabview', function (Y) {
             },
 
             /**
+             * The location being displayed
```

Comment:
can be confusing in the context of the "locations tab". Should rather be something like:
*The location being displayed in the location view*.